docs/icp/17-box-shadow-hover/17-box-shadow-hover.md

```diff
@@ -133,7 +135,276 @@ box-shadow: 5px 5px 5px 2px #808080;
 
 <img src="/icp/17/step-2.png" alt="step-2" width="500px"/>
 
-
 In this example, there is an image element with the class `img-tiger`. The CSS styles defined for the class `img-tiger` specify that the image should have a height of `300px`, a border-radius of `50px`, and a margin of `30px`. The `box-shadow` property is used to add a shadow to the image. The values `2px 2px 5px 10px` define the horizontal offset, vertical offset, blur radius, and spread radius of the shadow, respectively. The color of the shadow is specified as `rgb(138, 136, 136)`.
 
-When you view this HTML page in a browser, the image will have a box shadow applied to it according to the styles defined by the `.img-tiger` class. When you move your mouse over the image, the border-radius will change to 20px with a transition effect, as defined by the `.img-tiger:hover` selector.
\ No newline at end of file
+When you view this HTML page in a browser, the image will have a box shadow applied to it according to the styles defined by the `.img-tiger` class. When you move your mouse over the image, the border-radius will change to 20px with a transition effect, as defined by the `.img-tiger:hover` selector.
+
+## Background image
+
+The `background-image` property is used to set one or more background images.
+
+**Syntax :**
+
+```css
+background-image: url("");
+```
+
+**Example :**
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>background-image</title>
+    <style>
+      .container {
+        border: 1px solid black;
+        width: 600px;
+        height: 400px;
+        background-image: url("https://images.unsplash.com/photo-1579546929518-9e396f3cc809?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8fA%3D%3D&w=1000&q=80");
+      }
+    </style>
+  </head>
+
+  <body>
+    <div class="container">
+      <h1>This is a card</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio
+        voluptates cum veritatis odio voluptas nihil commodi sunt facere illum
+        quis nostrum nam quasi aut ipsam aliquam, similique accusamus nulla!
+        Molestias.
+      </p>
+    </div>
+  </body>
+</html>
+```
```

Comment:
add explanation of image like this 

In the above example, the `.container` class is used to define a container element with a background image. The `background-image` property is set to the URL of the desired image. In this case, we're using an image from Unsplash. The image will be displayed as the background of the container element.